tests/functional/requirements.txt

```diff
@@ -2,3 +2,4 @@ swagger-client==1.0.0
 pytest==6.1.2
 pydantic==1.9.1
 testcontainers==3.6.0
+docker==6.0.0
```

Comment:
Weird dependency. Do we need it?

Reply:
This is accepted.

Reply:
added this version of docker because of docker 4.0.0 Deprecation warnings in pytest. testcontainers depends on docker 4.0.0.